back/app/models/concerns/sluggable.rb

```diff
@@ -40,3 +41,7 @@ def slug_enabled?
     self.class.slug? && (!self.class.slug_if || self.class.slug_if&.call(self))
   end
 end
+
+def generate_fallback_slug(sluggable)
+  sluggable.id || SecureRandom.uuid
+end
```

Comment:
I'm not sure. I thought it might be desirable because I thought the UserSlugService did something similar, but I'm wrong. Not sure what made me think this, and maybe it's not a good idea to expose IDs (like user IDs). Shall I change it to always just return `SecureRandom.uuid`?